File: app/src/main/java/com/yuk/miuiHomeR/ui/FolderActivity.java
package com.yuk.miuiHomeR.ui;

import androidx.fragment.app.Fragment;

import com.yuk.miuiHomeR.R;
import com.yuk.miuiHomeR.ui.base.BaseAppCompatActivity;
import com.yuk.miuiHomeR.ui.base.SubFragment;

public class FolderActivity extends BaseAppCompatActivity {
    @Override
    public Fragment initFragment() {
        return new FolderFragment();
    }

    public static class FolderFragment extends SubFragment {

        @Override
        public int getContentResId() {
            return R.xml.prefs_folder;
        }

        @Override
        public void initPrefs() {

        }
    }
}

File: app/src/main/java/com/yuk/miuiHomeR/ui/FolderActivity.kt
package com.yuk.miuiHomeR.ui

import androidx.fragment.app.Fragment
import com.yuk.miuiHomeR.R
import com.yuk.miuiHomeR.ui.base.BaseAppCompatActivity
import com.yuk.miuiHomeR.ui.base.SubFragment
import com.yuk.miuiHomeR.utils.ktx.atLeastAndroidS
import moralnorm.preference.Preference

class FolderActivity : BaseAppCompatActivity() {

    override fun initFragment(): Fragment {
        return FolderFragment()
    }

    class FolderFragment : SubFragment() {
        override fun getContentResId(): Int {
            return R.xml.prefs_folder
        }

        override fun initPrefs() {
            val mSmallFolderBlurVisible = findPreference<Preference>("prefs_key_small_folder_blur")
            mSmallFolderBlurVisible.isVisible = atLeastAndroidS()
            mSmallFolderBlurVisible.isEnabled = mSmallFolderBlurVisible.isVisible
        }
    }
}